CondTools/Hcal/interface/instantiateEosTemplates.h

```diff
@@ -0,0 +1,74 @@
+#ifndef CondTools_Hcal_instantiateEosTemplates_h_
+#define CondTools_Hcal_instantiateEosTemplates_h_
+
+// Explicit instantiation of boost templates for eos archives.
+// Needed to overcome g++ quirks.
+#include <boost/archive/impl/basic_binary_oarchive.ipp>
+#include <boost/archive/impl/basic_binary_oprimitive.ipp>
+
+#if BOOST_VERSION < 104000
```

Comment:
@igv4321 - please start classes with a capital letter. Otherwise I don't see a reason to protect against boost version 1.40 (2009) as we are on 1.57